### ISSUE TITLE
 Inaccurate Accuracy

### ISSUE THREAD

**Issue by PratikHirapara** [maulikt04/Solar-Energy-Radiation-Prediction#1]
Currently i am replicating your project for my work.
could you please explain me how you got loss and accuracy?

#compiling the ANN:
ann.compile(optimizer = 'adam', loss = 'binary_crossentropy', metrics = ['accuracy'])
#Training the ANN on the Training Set:
ann.fit(x_train, y_train, batch_size = 20, epochs = 10)
Epoch 1/10
1144/1144 [==============================] - 2s 1ms/step - loss: 0.5477 - accuracy: 0.0000e+00
Epoch 2/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.4179 - accuracy: 0.0000e+00
Epoch 3/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.3908 - accuracy: 0.0000e+00
Epoch 4/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.3872 - accuracy: 0.0000e+00
Epoch 5/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.3869 - accuracy: 0.0000e+00
Epoch 6/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.3869 - accuracy: 0.0000e+00
Epoch 7/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.3869 - accuracy: 0.0000e+00
Epoch 8/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.3869 - accuracy: 0.0000e+00
Epoch 9/10
1144/1144 [==============================] - 2s 1ms/step - loss: 0.3869 - accuracy: 0.0000e+00
Epoch 10/10
1144/1144 [==============================] - 1s 1ms/step - loss: 0.3869 - accuracy: 0.0000e+00
<keras.callbacks.History at 0x2482fd35af0>

**Comment by maulikt04** [comment]
Yes I know about this problem. I am trying to solve this problem. If I will get solution then I will updated. 
If you have better idea about it then suggest me.
